flixel/addons/text/FlxTypeText.hx

```diff
@@ -44,6 +44,10 @@ class FlxTypeText extends FlxText
 	 */
 	public var cursorCharacter:String = "|";
 	/**
+	 * Set to true to prevent a word start typing in a line and jump to next line.
+	 */
+	public var preventWordJump:Bool = true;
```

Comment:
More of a line than a word jump I guess?

Any reason you'd wanna disable this?